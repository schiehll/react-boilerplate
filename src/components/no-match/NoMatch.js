import React from 'react'

const NoMatch = () => {
  return (
    <div>
      Ooops, we couldn't find this route.
    </div>
  )
}

export default NoMatch
