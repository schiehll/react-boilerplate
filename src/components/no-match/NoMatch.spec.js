import React from 'react'
import {render} from 'enzyme'
import {renderToJson} from 'enzyme-to-json'
import NoMatch from './NoMatch'

test('should render just fine', () => {
  const component = render(<NoMatch />)

  expect(renderToJson(component)).toMatchSnapshot()
})
