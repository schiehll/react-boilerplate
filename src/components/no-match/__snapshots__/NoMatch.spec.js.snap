exports[`test should render just fine 1`] = `
<div>
  Ooops, we couldn\'t find this route.
</div>
`;
